src/components/PluginsComponents/PluginsList.jsx

```diff
@@ -20,7 +20,6 @@ const PluginsList = () => (
             node {
               name
               version
-              downloads
```

Comment:
We no longer get downloads with the current registry call. We'd want to update the download script to grab this data as well but I think it should be a separate PR.